distributed/formats/csv.py

```diff
@@ -0,0 +1,89 @@
+import pandas as pd
```

Comment:
@jcrist the code in this file may interest you.  I'm retrying the `read_csv` problem with the approach laid out in https://github.com/dask/dask/issues/1022